isb_lib/core.py

```diff
@@ -500,7 +500,8 @@ def solr_max_source_updated_time(
         dict = res.json()
         docs = dict["response"]["docs"]
         if docs is not None and len(docs) > 0:
-            return dateparser.parse(docs[0]["sourceUpdatedTime"])
+            # Strip out the time zone since that is confusing matters
+            return dateparser.parse(docs[0]["sourceUpdatedTime"]).replace(tzinfo=None)
```

Comment:
This is the line that feels dirty.

Reply:
Per discussion this morning in standup, the bug was actually in the postgresql instance on my local machine.  It was configured to use the Arizona timezone.  When I changed my local server to run in UTC, everything worked fine.